developers/contributor-guide/weaviate-core/tests.md

```diff
@@ -87,29 +87,29 @@ go test ./...
 #### Adding new unit tests
 
 * Add unit tests in the same folder as the code they are testing
-* Aim to write "black box" unit tests which test the public ("exported") methods of the class under test without knowing too much about the internals
+* Aim to write "black box" unit tests that test the public ("exported") methods of the class under test without knowing too much about the internals.
 * Do not use any build tags.
 
 ### Integration tests
-As outline in the Philosophy, integration tests require backing services to be run. We have a convenience script available which starts all required services in `Docker Compose` and runs the tests against it:
+As outline in the Philosophy, integration tests require backing services to be run. We have a convenience script available which starts all required services in `docker compose` and runs the tests against it:
 
 ```sh
 test/integration/run.sh
 ```
 
 #### Adding new integration tests
 
-* Use the `integrationTest` build tag on your test, so it is ignored during unit test runs
-* Make sure the test prepares for and cleans up after itself, so tests can be run succession
-* If your test requires a lot of time to execute, consider marking them as a slow test and making them optional. (see below)
+* Use the `integrationTest` build tag on your test, so it is ignored during unit test runs.
+* Make sure the test prepares for and cleans up after itself, so tests can be run in succession.
+* If your test requires a lot of time to execute, consider marking them as a slow test and making them optional. (see below).
```

Comment:
This are pre-existing errors. Would you mind updating with your changes?

consider marking it
and making it